### ISSUE TITLE
Host header is missing from the Headers section

### ISSUE THREAD

**Issue by sashee** [darklynx/request-baskets#61]
When I send a request with a different Host header then the request is logged successfully but the Host is missing.

```
curl -H "Host: example.com" https://rbaskets.in/3vgdn65
```

Shows:

![image](https://user-images.githubusercontent.com/82075/89011417-9b7fc180-d310-11ea-9c0a-1abf49cfad2a.png)

Testing with webhooks.site the host header is listed:

![image](https://user-images.githubusercontent.com/82075/89011578-de419980-d310-11ea-9a3f-bee5ace8dc41.png)

This prevents debugging host-related issues.

**Comment by darklynx** [comment]
This is a result of Go-lang `net/http` library behavior, see https://golang.org/src/net/http/request.go#L159

> // For incoming requests, the Host header is promoted to the
> // Request.Host field and removed from the Header map.

:( 

On the other hand the `Request.Host` field does not contain a host header only instead it is defined as following: https://golang.org/src/net/http/request.go#L221

**TL;DR**: if `Host`-header is not defined, `Request.Host` will contain a host part of the URL the request was made to :( 

In order to expose `Host`-header properly I need to find out how to either get an access to the raw HTTP-request, or how to figure out that `Request.Host` contains the host hedaer
